src/main/java/com/n1analytics/paillier/EncryptedNumber.java

```diff
@@ -416,4 +416,28 @@ public boolean equals(EncryptedNumber o) {
             context.equals(o.context) &&
             ciphertext.equals(o.ciphertext));
   }
+
+  /**
+   * Return a byte array representing the underlying raw number.
+   *
+   * WARNING: This method may return unencrypted information! Under no
+   * circumstances should the return value be sent anywhere.
+   *
+   * WARNING: This method is not intended for general use and may be
+   * removed in a future release. Think twice before using it.
+   */
+  public byte[] toByteArrayUnsafe() {
+    return ciphertext.toByteArray();
+  }
+
+  /**
+   * Return true or false according to whether this EncryptedNumber is
+   * safe for transmission.
+   *
+   * WARNING: This method is not intended for general use and may be
+   * removed in a future release. Think twice before using it.
+   */
+  public boolean getIsSafe() {
```

Comment:
So would I, but the variable is already called _isSafe_, so there'd be a name conflict.